homeassistant/components/kitchen_sink/__init__.py

```diff
@@ -110,6 +119,23 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     # Notify backup listeners
     hass.async_create_task(_notify_backup_listeners(hass), eager_start=False)
 
+    # Subscribe to labs feature updates for kitchen_sink preview repair
+    @callback
+    def _async_labs_updated(event: Event[EventLabsUpdatedData]) -> None:
+        """Handle labs feature update event."""
+        if (
+            event.data["domain"] == "kitchen_sink"
```

Comment:
Can we use `DOMAIN` const here?

Reply:
oh sorry the domain constant using kitchen_sink; yeah sure 👍